api/metrics/views.py

```diff
@@ -151,3 +152,33 @@ def metric_type(self):
     @property
     def metric(self):
         return PreprintDownload
+
+class InstitutionSummaryMetrics(JSONAPIBaseView):
+    # TODO: Identify necessary permissions level for InstitutionSummaryMetrics
```

Comment:
I think we'd be safe with guardian permissions [like so](https://github.com/CenterForOpenScience/osf.io/pull/9318/files#diff-3b3e64edb1e8e2455d8c6c9ba66efe21R153-R164)